CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
  ### React Addons
   
-* Remove PropTypes dependency from ReactLink. ([@gaearon](https://github.com/gaearon) in [#9766](https://github.com/facebook/react/pull/9766))
-* Fix issue where environment variable was not being transformed by browserify. ([@mridgway](https://github.com/mridgway) in [#9642](https://github.com/facebook/react/pull/9642))
```

Comment:
The first one is still there (I just moved it).

The second one shouldn't be there. It was cherry picked to 15.5.x as an urgent fixed so I already added it to relevant patch releases of addons in the changelog a few weeks ago.